module1/requirements/promotion_requirements.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Promotion to Module 2
+layout: page
+---
+
+## Learning Goals
+
+Promoted students will demonstrate proficiency in the material covered in Module 1.
+
+## Projects
+
+You will complete four [projects](../projects/) throughout Module 1, each of which will be assessed according to a rubric. A passing project will "Meet Expectations" (per the rubric) for most categories. You must submit every project to be eligible for promotion.
+
+* Week 1-2: Solo Project #1
+  * The Solo Project is evaluated, but does not count toward promotion
+* Week 2-3: Solo Project #2
+* Week 3-4: Paired Project
+  * You must pass EITHER the Paired Project OR Solo Project #2
+* Week 5-6: Group Project
+  * You must pass the Final Project
+  * No rubric category can score Below Expectations
+
+## Assessments and Independent Challenges
+
+You will complete two assessments and three weekly Independent Challenges in Module 1, each of which will be assesed on a scale of Below Expectations, Approaching Expectations, Meets Expectations, and Exceeds Expectations. Meets Expectations is considered a passing score.
+
+### Assessments
+1. Mid Module Assessment (aka "Mid Mod")
+  * The Mid Mod is evaluated, but does not count toward promotion
+
+1. Final Independent Challenge (aka "Final IC", Week 6)
+  In order to be promoted to Module 2:
+  * You must earn a _Meets Expectations_ on the Week 6 Final IC to be promoted to Module 2. Students who do not will be offered a re-take.
+
+### Independent Challenges
+* Weekly Independent Challenges (ICs)
+  * Weeks 2, 4, 5 (aka "Week 2 IC", "Week 4 IC", etc.)
```

Comment:
Not sure if this is changing, but currently in Mod 1, the ICs are only done weeks 3, 4(optional), and 5.

Reply:
Ooh Thank you! Fixing now and merging